### DISCUSSION TITLE
Integrate Windows night mode (low blue light) into taskbar popup

### DISCUSSION THREAD

**Discussion by Linx-ESP** [xanderfrangos/twinkle-tray#586] [Ideas]
Adding a button in the popup would make for a fast way to turn down the brightness and setting night mode, useful at night.

**Comment by yurkovych** [comment]
Came to suggest this. All night mode apps like f.lux are scheduled and overcomplicated, to my mind. Integrated Windows Night Mode adjustment is just what I need, but you need to dig deep every time you wish to slightly adjust the light temperature of the monitor. If there was a simple slider in Twinkle Tray to adjust night light intensity, it would be amazing. Cheers!

**Comment by flow23** [comment]
Same here ...
